packages/xo-web/src/common/store/reducer.js

```diff
@@ -86,6 +86,11 @@ export default {
     }
   ),
 
+  // These Ids are used temporary to be preselected in backup-ng/new/vms
+  tmpVmIds: combineActionHandlers([], {
```

Comment:
`homeVmIdsSelection` to know where this ids come from?

Reply:
:heavy_check_mark: fixed